flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/metadata/FlinkRelMdColumnInterval.scala

```diff
@@ -565,7 +566,10 @@ class FlinkRelMdColumnInterval private extends MetadataHandler[ColumnInterval] {
           inputType: RelDataType,
           isBounded: Boolean): AggregateCall = {
         val outputIndexToAggCallIndexMap = AggregateUtil.getOutputIndexToAggCallIndexMap(
-          aggCalls, inputType, isBounded)
+          ShortcutUtils.unwrapTypeFactory(input),
```

Comment:
how about we do a static import for all `ShortcutUtils.unwrapTypeFactory` to make the code more concise. I added a similar feedback to Marios PR recently.